src/main/java/org/codehaus/mojo/license/download/LicenseDownloader.java

```diff
@@ -104,7 +104,7 @@ public LicenseDownloader( Proxy proxy, int connectTimeout, int socketTimeout, in
             configBuilder.setProxy( new HttpHost( proxy.getHost(), proxy.getPort(), proxy.getProtocol() ) );
         }
 
-        HttpClientBuilder clientBuilder = HttpClients.custom().setDefaultRequestConfig( configBuilder.build() );
+        HttpClientBuilder clientBuilder = HttpClients.custom().setCookieSpec(CookieSpecs.STANDARD).setDefaultRequestConfig( configBuilder.build() );
```

Comment:
There seems to be a missing import. Did you actually test-compile it locally?
Please run `mvn verify` locally.